lib/msal-browser/src/controllers/NestedAppAuthController.ts

```diff
@@ -367,7 +369,16 @@ export class NestedAppAuthController implements IController {
             nestedAppAuthRequest: true,
         });
 
-        const result = await this.acquireTokenFromCacheInternal(request);
+        // respect cache lookup policy
+        let result: AuthenticationResult | null = null;
+        switch (request.cacheLookupPolicy) {
+            case CacheLookupPolicy.Default:
+            case CacheLookupPolicy.AccessToken:
```

Comment:
Should this also include `AccessTokenAndRefreshToken`?

Reply:
I am debating it. NAA i.e., the embedded app will not have RT in cache. We can have the parent anyway fetch based on the cache policy.

Reply:
I think that's still ok if the `RefreshToken` part of it fails but it seems wrong to return null here if `AccessToken` part is specified.

Alternatively, maybe don't allow that policy at all for NAA and throw if provided? Up to you.

Reply:
I think adding it is needed. The RT part of the cache policy will be respected by the parent. Added and updated tests. Please check.

Reply:
We cross-wired our responses. Made the needed changes, please check.

Reply:
Thanks!